samples/ExampleGallery/UAP/CameraEffectExample.xaml

```diff
@@ -27,5 +27,8 @@ Licensed under the MIT License. See LICENSE.txt in the project root for license
         </StackPanel>
         <ComboBox x:Name="effectCombo" ItemsSource="{Binding ElementName=control, Path=PossibleEffects}"
                   SelectionChanged="EffectCombo_SelectionChanged" />
+        <Slider x:Name="blurAmountSlider" Header="Blur" Minimum="0" Value="3" Maximum="12" Visibility="Visible" Orientation="Vertical" HorizontalAlignment="Right"
```

Comment:
Slider control to change Blur intensity of new effect. Is visible only when the BlurEffect is active.